app/components/application-wrapper.js

```diff
@@ -56,20 +55,17 @@ export default Component.extend({
 
 		return `${vertical}-vertical`;
 	}),
-	// Smart Banner AB Testing
+
 	/**
 	 * @returns {boolean}
 	 */
-	shouldShowABTestBannerOnIOS: computed('currentUser.language', function () {
-		return system === 'ios' &&
-			this.get('currentUser.language') === 'en' &&
-			inGroup('FANDOM_APP_SMART_BANNER_IOS_EXPERIMENT', 'BANNER');
+	shouldShowFandomAppSmartBanner: computed('currentUser.language', 'wikiVariables', function () {
+		return this.get('currentUser.language') === 'en' && this.get('wikiVariables.enableFandomAppSmartBanner');
 	}),
 
-	isIosBannerVisible: computed('shouldShowABTestBannerOnIOS', 'smartBannerVisible', function () {
-		return this.get('shouldShowABTestBannerOnIOS') && this.get('smartBannerVisible');
+	isFandomAppSmartBannerVisible: computed('shouldShowFandomAppSmartBanner', 'smartBannerVisible', function () {
```

Comment:
https://www.emberjs.com/api/ember/2.15/namespaces/Ember.computed/methods/and?anchor=and